portx/modules/ROOT/pages/_partials/relationships-settings.adoc

```diff
@@ -0,0 +1,7 @@
+. xref:index.adoc#start-portx[Start PortX].
```

Comment:
Above the first step, add something like this: 
"Configuration is at the **_Relationship_** level.
- For information about configuring Relationships, see [insert ref to Relationships config section ].
- For more information about Relationships. see [Actors, Relationships and Artifacts](https://docs.portx.io/integration-hub/latest/actors-relationships-and-artifacts.html)."

Editorial Notes (for you):
* Bold italics for the first occurrence of Relationship, and link it to the letter **R** in the Glossary (see other glossary terms for examples of how to do this).
* Capitalize Relationship because, as we use it in a way that's specific to us, it's a proper noun.
* It appears we do not have a Relationships config page in the docs! So we must create that before we can link to it. 
* Use the xref method to link to the _Actors_ et al. Page, not the http: method I put there. 
* For your own education, check out that page and also the different scenarios under Routing Scenarios. There's a lot of useful info about how Relationships work in there.